src/clusterd/src/bin/clusterd.rs

```diff
@@ -85,21 +84,6 @@ struct Args {
         default_value = "http://localhost:6879"
     )]
     persist_pubsub_url: String,
-    /// Whether to use the new txn-wal tables implementation or the legacy
-    /// one.
-    ///
-    /// This flag is only used to force clusterd restarts when the value in
-    /// environmentd is changed.
-    ///
-    /// This flag is named "persist-txn-tables" and not "txn-wal-tables" for
-    /// historical reasons.
-    #[clap(
-        long,
-        env = "PERSIST_TXN_TABLES",
-        default_value = "off",
-        parse(try_from_str)
-    )]
-    persist_txn_tables: TxnWalTablesImpl,
```

Comment:
note to self: it's safe to remove this because the only thing that starts up these processes is envd running the same version of code